Index: test_main.http
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Test your FastAPI endpoints\r\n\r\nGET http://127.0.0.1:8000/\r\nAccept: application/json\r\n\r\n###\r\n\r\nGET http://127.0.0.1:8000/hello/User\r\nAccept: application/json\r\n\r\n###\r\n\r\nPOST http://127.0.0.1:8000/incomes\r\nAccept: application/json\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/test_main.http b/test_main.http
--- a/test_main.http	
+++ b/test_main.http	
@@ -5,7 +5,7 @@
 
 ###
 
-GET http://127.0.0.1:8000/hello/User
+GET http://127.0.0.1:8000/incomes
 Accept: application/json
 
 ###
